{{cookiecutter.project_name}}/monitoring/model_monitoring_setup_template.py

```diff
@@ -0,0 +1,278 @@
+{%raw%}
+def get_notebook(timestamp_col, model_version_col, prediction_col, problem_type, table_name, granularity, model_name):
+    return f'''# Databricks notebook source
+# Install data monitoring client library. Eventually this will be included in MLR
+%pip install "https://databricks-mvn.s3.amazonaws.com/databricks-data-monitoring/databricks-data-monitoring-buZrBwyN.tar.gz?AWSAccessKeyId=AKIAJLF6DMKWZKIYMPAQ&Expires=1990213870&Signature=JouwkAylFhuviaVDArkznIZQkgM%3D"
+
+# COMMAND ----------
+
+# import libraries
+from pyspark.sql import types as T
+import databricks.data_monitoring as dm
+from databricks.data_monitoring.analysis import analysis_type
+from pyspark.sql.functions import col,to_timestamp
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Required Fields
+
+# COMMAND ----------
+
+# The name of the column that represents the inference timestamp (i.e. time of prediction) for this row. This column should be of TimestampType.
+timestamp_col = "{timestamp_col}"
+
+# The name of the column that represents the model version used to predict on this row. 
+# This should correspond to the MLflow registered model version (or any identifier unique to a specific trained model).
+model_version_col = "{model_version_col}"
+
+# The name of the column that holds the prediction (from inference) for this row.
+prediction_col = "{prediction_col}"
+
+# The type of ML problem that the model is solving. This should be either “classification” or “regression”.
+# Other problem types are not supported at this time. 
+problem_type = "{problem_type}"
+
+# The inference table holding data to be monitored
+table_name = "{table_name}"
+
+# List of granularities to use when aggregating data
+# into time windows based on their timestamp. Currently the following static granularities are supported:
+# {{"5 minutes", "30 minutes", "1 hour", "1 day", "n week(s)", "1 month", "1 year"}}.
+# For tables that does not have timestamp column (e.g. Generic), the `granularities` must be set as None,
+# which means that time-aggregation is based on the time of calling `refresh_metrics`.
+granularities = ["{granularity}"]
+
+# The model to be linked by the monitoring
+model_name = "{model_name}"
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Optional Fields
```

Comment:
Do you expect users to edit this notebook directly? If not, why not make them optional parameters to your script? I'm guessing users will want to control things like output schema, or provide a baseline, or provide labels so they can have model quality metrics.